habitat_baselines/evaluate_ppo.py

```diff
@@ -198,8 +198,13 @@ def eval_checkpoint(checkpoint_path, args, writer, cur_ckpt_idx=0):
                 episode_stats["success"] = int(infos[i]["spl"] > 0)
                 episode_stats["reward"] = current_episode_reward[i].item()
                 current_episode_reward[i] = 0
-                stats_episodes[current_episodes[i].episode_id] = episode_stats
-
+                # use scene_id + episode_id as unique id for storing stats
+                stats_episodes[
+                    "{}:{}".format(
```

Comment:
Tuples can be keys to dictionaries, that is probably cleaner :)

Reply:
Also the logic of deciding if an environment should be paused or not needs to use this key.

Reply:
> Also the logic of deciding if an environment should be paused or not needs to use this key.

Whoops. Should be fixed now :)